.github/workflows/zcashd-nightly.yml

```diff
@@ -75,21 +75,23 @@ jobs:
           ./zcash/fetch-params.sh
       - name: Run ziggurat suite
         continue-on-error: true
-        env:
-          FILENAME: ${{ github.event.repository.updated_at }}
         run: |
           rm ./ziggurat/*.d
           mv ./ziggurat/ziggurat-* ziggurat_test
           chmod +x ziggurat_test
           chmod +x zcash/zcashd
-          mkdir -p results
-          ./ziggurat_test --test-threads=1 --nocapture -Z unstable-options --report-time --format json > results/$FILENAME.jsonl
-          rm -r results/latest.jsonl
-          cp results/$FILENAME.jsonl results/latest.jsonl
+          mkdir -p results/zcashd
+          ./ziggurat_test --test-threads=1 --nocapture -Z unstable-options --report-time --format json > result.jsonl
       - name: git
+        env:
+          FILENAME: ${{ github.event.repository.updated_at }}
```

Comment:
will `updated_at` return the time of the current run, or the time at which the repo was last updated aka "latest - 1"?

Reply:
Good question. I'd assume it's the time of the most recent commit, in which case we'll need to find another solution when migrating from trigger-on-push to a cron-based one.